processor/file.go

```diff
@@ -66,7 +66,10 @@ func NewDirectoryWalker(output chan<- *FileJob) *DirectoryWalker {
 		output: output,
 	}
 	for _, exclude := range Exclude {
-		directoryWalker.excludes = append(directoryWalker.excludes, regexp.MustCompile(exclude))
+		regexpResult, err := regexp.Compile(exclude)
+		if err == nil {
+			directoryWalker.excludes = append(directoryWalker.excludes, regexpResult)
+		}
```

Comment:
Can you put a condition on there to handle if there is an error,

```
} else {
    printError(err.Error())
}
```

Then at least its explicitly listed why it didn't work as expected, but without interrupting the run itself.

Reply:
sure, code updated, plz check pr again